python/cm/issue.py

```diff
@@ -10,135 +10,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from collections import defaultdict
-
-import cm.status_api
 from cm.adcm_config import proto_ref, obj_ref, get_prototype_config
 from cm.errors import AdcmEx, raise_AdcmEx as err
 from cm.hierarchy import Tree
```

Comment:
I think Tree is too generic name for this class. Maybe something more specific like ClusterTree?